frontend/src/app/lightning/nodes-per-isp-chart/nodes-per-isp-chart.component.ts

```diff
@@ -70,8 +70,12 @@ export class NodesPerISPChartComponent implements OnInit {
                   data.ispRanking[i][5] = i;
                 }
                 for (let i = 0; i < data.ispRanking.length; ++i) {
-                  data.ispRanking[i][6] = Math.round(data.ispRanking[i][4] / nodeCount * 10000) / 100;
-                  data.ispRanking[i][7] = Math.round(data.ispRanking[i][2] / totalCapacity * 10000) / 100;
+                  let roundingFactor = 100;
+                  if (data.ispRanking[i][1] === 'wiz') {
```

Comment:
We should not hard code anything to show up like that.

Reply:
Yeah this was not what I meant lol, just change the 0.5% to 0.4% or something